### PR TITLE
Update 4.3.7_smartproxy_puppet.md

### PR DESCRIPTION
**Pull request by LordBrain**
Added sudoers rule for puppetssh

### CODE REVIEW

File: _includes/manuals/1.7/4.3.7_smartproxy_puppet.md [hunk 1]
```diff
@@ -112,6 +112,11 @@ The puppetssh provider uses SSH to connect to the client using SSH keys and run
 #:puppetssh_keyfile: /etc/foreman-proxy/id_rsa
 </pre>
 
+And then add a sudoers rule:
```
Comment (domcleal) [inline _includes/manuals/1.7/4.3.7_smartproxy_puppet.md:115]: perhaps add "if puppetssh_sudo is enabled"?  (I assume it won't sudo to root otherwise)

**Comment by mmoll** [comment]
@blee1170 ping? could you add this to the 1.8 and 1.9 docs?

**Comment by mmoll** [comment]
I'm leaning towards closing this as foreman-installer should be used anyway for that configuration. If wanted I'll open a PR to newer versions.

**Comment by mmoll** [comment]
Closing this as there was no reaction. As already mentioned, foreman-installer should be used anyway. If anybody wants to pick this up, the howto of #451 would also be in need of an addition then.

**Pull request closed without merging**